book_maker/translator/chatgptapi_translator.py

```
@@ -189,20 +223,26 @@ def translate(self, text, needprint=True):
                 # 1. openai server error or own network interruption, sleep for a fixed time
                 # 2. an apikey has no money or reach limit, don`t sleep, just replace it with another apikey
                 # 3. all apikey reach limit, then use current sleep
-                sleep_time = int(60 / self.key_len)
+                sleep_time = int(10 / self.key_len)
                 print(e, f"will sleep {sleep_time} seconds")
                 time.sleep(sleep_time)
                 attempt_count += 1
                 if attempt_count == max_attempts:
                     print(f"Get {attempt_count} consecutive exceptions")
                     raise
             except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)